index.html

```diff
@@ -114,6 +153,14 @@ <h1>HAVE QUESTIONS?<br>
             </form>
         </div>
     </div>
+    <div class="footer-tablet">
```

Comment:
Please don't create tablet version of divs, do not repeat code, just style previous one with css.